tests/syncoid/run-tests.sh

```diff
@@ -2,7 +2,7 @@
 
 # run's all the available tests
 
-for test in */; do
+for test in $(find . -mindepth 1 -maxdepth 1 -type d -printf "%P\n" | sort -g); do
```

Comment:
not needed with the numbering scheme

Reply:
[Commit dropped](https://github.com/jimsalterjrs/sanoid/compare/3443717b3b95f4cee62880b44b39ac067989c755..3d0571fe02204bffd9d9d41d05d31fcb71f1ec75)